src/v/v8_engine/api.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/vectorizedio/redpanda/blob/master/licenses/rcl.md
+ */
+
+#pragma once
+
+#include "v8_engine/environment.h"
+#include "v8_engine/executor.h"
+
+#include <seastar/core/sharded.hh>
+
+namespace v8_engine {
+
+// This class implements wrapper for executor.
+// In current design executor exist on one core
+// And for submit tasks you need to run invoke_on
+// So this class wrap this logic for v8_engine::script
+class executor_service {
+public:
+    executor_service(
+      ss::alien::instance& instance, unsigned core, int64_t size);
+
+    ss::future<> start();
+    ss::future<> stop();
+
+    // We do not need use gate there, because executor has gate inside submit
+    // function
+    template<typename WrapperFuncForExecutor>
+    ss::future<> submit(
+      WrapperFuncForExecutor&& func_for_executor,
+      std::chrono::milliseconds timeout) {
+        return _executor.invoke_on(
+          0,
```

Comment:
magic number make global constant

Reply:
No
_core - is core for std::thread, 0 - is seastar core with executor service.On first step it will be same, but in the future I believe executor will be changed and it will means different things